### ISSUE TITLE
Does anyone successfully train this model?

### ISSUE THREAD

**Issue by hiwonjoon** [ry/tensorflow-resnet#15]
Hi, I am trying to use this model for my specific dataset, but it gives a NaN after few steps(no more than 10 steps). At the beginning, the loss value seems right but, all of sudden, it goes NaN in a single step, like from 5.5 to NaN. I tried extremely small learning rate, but the result was same. 

It seems that there are division by zero. FYI, I am using tf-0.10-rc0. Any ideas?
